lib/GithubHandler.js

```diff
@@ -177,15 +177,15 @@ GithubHandler.prototype.pullRequestHandler = function(event, cb) {
     type: 'pull_request', // also in event.event
     service: 'github',
     host: undefined,
+    branch: event.payload.pull_request.head.ref,
```

Comment:
Right, that all makes sense. What I was looking for was that each pull request is a request to merge some branch "source" into some branch "target" - I was wondering if we'd maybe want to capture that somewhere.

I also wonder if there's a better way to do that merge that wouldn't put us at risk of a race condition.  If your build takes a couple of minutes to start (after we have queuing) and you push an update we could end up performing two builds of the newer change because when the older one runs the PR may have already been updated.